lib/core/card-view/models/card-view-baseitem.model.ts

```diff
@@ -30,7 +30,7 @@ export abstract class CardViewBaseItemModel {
 
     constructor(cardViewItemProperties: CardViewItemProperties) {
         this.label = cardViewItemProperties.label || '';
-        this.value = cardViewItemProperties.value;
+        this.value = cardViewItemProperties.value && cardViewItemProperties.value.displayName || cardViewItemProperties.value;
```

Comment:
if displayName is a falsy value we have the same scenario that raised this bug in the first place 
`... || cardViewItemProperties.value;`

Reply:
true, indeed it is not the perfect solution, but it is an improvement that fixes the mentioned bug.

Reply:
I don't consider it an improvement since it's too specific and defaults to the same bug

Reply:
let's take it one step at a time: fix this specific case and I'll open bug on ADF so to find the fix of the general case.

Reply:
https://issues.alfresco.com/jira/browse/ADF-4700 - created.